stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2963,8 +2963,8 @@ internal enum _Variant${Self}Storage<${TypeParametersDecl}> : _HashStorage {
   internal typealias Value = ${TypeParameters}
 %end
 
-  case native(NativeStorageOwner)
   case cocoa(CocoaStorage)
+  case native(NativeStorageOwner)
```

Comment:
It looks like we rely on Variant{Dictionary,Set}Storage.native having a tag value of 0, and .cocoa having a tag value of 1. With this change, this is swapped around.